k8s/mock_app_cr.yaml

```diff
@@ -8,4 +8,5 @@ metadata:
     app: warden
 spec:
   replicas: 1
```

Comment:
If I remember correctly, it is better to not set `replicas` when there is auto scaling to avoid issues with upgrading. When you are upgrading the resource like this it forces `1` replica and it needs to be scaled up again (for short amount of time there are less replicas)